### PR TITLE
Update Set-SPOBuiltInSiteTemplateSettings.md

### PR DESCRIPTION
**Pull request by ansilv**
Added missing Template IDs from Feature ID 85659 https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=&searchterms=85659

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOBuiltInSiteTemplateSettings.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ All site templates are displayed by default.
 | Training and courses        | 695e52c9-8af7-4bd3-b7a5-46aca95e1c7e  | 
 | Training and development team     | 64aaa31e-7a1e-4337-b646-0b700aa9a52c | 
 | Team collaboration     | 6b96e7b1-035f-430b-92ca-31511c51ca72  | 
+| Retail Management     | e4ec393e-da09-4816-b6b2-195393656edd  |
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOBuiltInSiteTemplateSettings.md:38]:
```suggestion
| Retail Management     | e4ec393e-da09-4816-b6b2-195393656edd  | 
```
```suggestion
| Retail management     | e4ec393e-da09-4816-b6b2-195393656edd  | 
```

Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOBuiltInSiteTemplateSettings.md:38]:
```suggestion
| Retail management     | e4ec393e-da09-4816-b6b2-195393656edd  | 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ansilv, kindly commit the suggested minor change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]
For Healthcare and store collaboration, I'm receiving "This is not a valid site template ID" using PS module 16.0.21812.0. Will they appear in the next release?

**Comment by ansilv** [comment]
> For Healthcare and store collaboration, I'm receiving "This is not a valid site template ID" using PS module 16.0.21812.0. Will they appear in the next release?

I think they are not in Production yet, at least they don't show in the templates for me. However, they should be released soon (?) and I got the IDs from the code, same place I got the retail management from: /odsp-next/apps/odsp-next/src/odsp-next/controls/webTemplatesGallery/components/WebTemplatesGalleryConstants.ts

https://onedrive.visualstudio.com/ODSP-Web/_git/odsp-web?path=/odsp-next/apps/odsp-next/src/odsp-next/controls/webTemplatesGallery/components/WebTemplatesGalleryConstants.ts&_a=contents&version=GBmain

**Comment by dariomws** [comment]
Thanks @ansilv!

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**